src/OpenFeature.SDK/OpenFeatureClient.cs

```diff
@@ -209,6 +209,7 @@ public FeatureClient(IFeatureProvider featureProvider, string name, string versi
                 .Concat(OpenFeature.Instance.GetHooks())
                 .Concat(this._hooks)
                 .Concat(options?.Hooks ?? Enumerable.Empty<Hook>())
+                .Concat(this._featureProvider.GetProviderHooks())

```

Comment:
I think the merging works as expected here (basically, hooks win... see: https://github.com/open-feature/spec/blob/main/specification/sections/04-hooks.md#requirement-434)

But I _don't_ see a test that specifically asserts this behavior (let me know if I'm wrong). If I'm correct, and it's alright with you @benjiro , I will write this test myself just to make sure I can get this project setup fully locally (I've pulled it but never run the tests suite).

Reply:
Yea go ahead and update the test case. Let me know if you have any issues I've only tested the project via command line, Rider and VS. if you're using vscode you might need to fix up the git ignores or do some initial setup. If you run into any troubles ping me on slack happy to help

Reply:
I'm going to branch of this branch to make this change, feel free to merge or not.